.github/workflows/gen_json.yml

```
@@ -0,0 +1,46 @@
+name: Test API JSON generator
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
```

Amazing idea!

Try

```suggestion
on:
  push:
  pull_request:
```

---

go ahead and change it

---

The master needs to be there because the generator is not available on other branches.

---

we can always change that later.

I will adjust it.

---

this line also might be causing an issue..

```yml
    if: github.repository == 'lvgl/lvgl'

```

---

still not showing up in...

---

dammit I keep on forgetting to tag you. @liamHowatt  If I don't do that you don't get a notification